src/C0_data_processing.py

```diff
@@ -216,6 +216,37 @@ def energyConversion(dict_values, group):
             dict_values["economic_data"],
             dict_values[group][asset],
         )
+        if "maximumCap" in dict_values[group][asset]:
+            # check if maximumCap is greater that installedCap
+            if dict_values[group][asset]["maximumCap"]["value"] is not None:
+                if (
+                    dict_values[group][asset]["maximumCap"]["value"]
+                    < dict_values[group][asset]["installedCap"]["value"]
+                ):
+
+                    logging.warning(
+                        "The stated maximumCap in %s %s is smaller than the "
+                        "installedCap. Please enter a greater maximumCap."
+                        "For this simulation, the maximumCap will be "
+                        "disregarded and not be used in the simulation",
+                        group,
+                        asset,
+                    )
+                    dict_values[group][asset]["maximumCap"]["value"] = None
+                # check if maximumCao is 0
+                elif dict_values[group][asset]["maximumCap"]["value"] == 0:
+                    logging.warning(
+                        "The stated maximumCap of zero in %s %s is invalid."
+                        "For this simulation, the maximumCap will be "
+                        "disregarded and not be used in the simulation.",
+                        group,
+                        asset,
+                    )
+                    dict_values[group][asset]["maximumCap"]["value"] = None
+        else:
+            dict_values[group][asset].update(
+                {"maximumCap": {"value": None, "unit": "kWp"}}
```

Comment:
```suggestion
                {"maximumCap": {"value": None, "unit": dict_values[group][asset]["unit"]}}
```
I think this should work (please test once). If left as above, it is kWp even for transformers.